SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -160,6 +160,26 @@ SimpleNumber : Number {
 		}
 	}
 
+	snap { arg resolution = 1.0, tolerance = 0.05, strength = 1.0;
```

Comment:
i would question the name "tolerance" for the second parameter. usually, that indicates a range of untransformed values, but here it's actually describing the range of effect. i searched around for a name for this, and all i could come up with was "margin". what do you think?

Reply:
I like margin as it provides a good visualization of what's going on. I shall change.